lib/token.js

```diff
@@ -15,17 +15,17 @@ module.exports = token
 
 token.usage =
   'npm token list\n' +
-  'npm token delete <tokenKey>\n' +
+  'npm token revoke <tokenKey>\n' +
   'npm token create [--read-only] [--cidr=list]\n'
 
-token.subcommands = ['list', 'delete', 'create']
+token.subcommands = ['list', 'revoke', 'create']
 
 token.completion = function (opts, cb) {
   var argv = opts.conf.argv.remain
 
   switch (argv[2]) {
     case 'list':
-    case 'delete':
+    case 'revoke':
```

Comment:
I think you should accept both here. This would be a breaking change otherwise, and I think it's fine to have `delete` as an alias.

Reply:
oh oops, I didn't mean to ditch it entirely!

Reply:
ah, this is tab completion. I'm ok with not tab completing delete.